README.md

```diff
@@ -50,6 +50,33 @@ A simple library to generate and retrieve OAuth2 tokens for use with Google Clou
     }
     ```
 
+## Google Compute Metadata
+
+Every compute instance stores its metadata on a metadata server.
+Goth can query this metadata server to fetch authentication credentials
+for a service account within the instance.
+
+To query the metadata server for an access token, you must configure
+the following:
+
+  * `url` must be set to the base url for the metadata server.
+  * `credentials` must be set to a tuple `{:instance, String.t()}`
```

Comment:
I think we should _name_ the tuple element, we can later see it is a string in the example, wdyt?

```suggestion
  * `credentials` must be set to a tuple `{:instance, account}`
```